runtime/src/lib.rs

```diff
@@ -913,34 +864,4 @@ mod tests {
 		is_submit_signed_transaction::<SubmitTransaction>();
 		is_sign_and_submit_transaction::<SubmitTransaction>();
 	}
-
-	#[test]
-	fn block_hooks_weight_should_not_exceed_limits() {
```

Comment:
Test got removed upstream after some changes to block weight involving `on_initialize` and `on_finalize`